packages/ui/src/components/SidebarItem/SidebarItem.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+
+import { OutlinedDashboardIcon, SolidDashboardIcon } from '../../icons';
+import { SidebarItem } from './SidebarItem';
+
+describe('SidebarItem', () => {
+  it('renders correctly', () => {
+    render(
+      <SidebarItem path="/" icon={<OutlinedDashboardIcon />} iconSelected={<SolidDashboardIcon />} count={2021}>
+        Dashboard
+      </SidebarItem>,
+      { wrapper: MemoryRouter },
+    );
+
+    const sidebarItem = screen.getByRole('group');
+    const sidebarItemIcon = screen.getByLabelText(/dashboard/i);
+    const sidebarItemBadge = screen.getByText(2021);
+
+    expect(sidebarItem).toBeInTheDocument();
+    expect(sidebarItemIcon).toBeInTheDocument();
+    expect(sidebarItemBadge).toBeInTheDocument();
+  });
+
+  it('renders correctly in disabled state', () => {
+    render(
+      <SidebarItem path="/" disabled>
+        Dashboard
+      </SidebarItem>,
+      { wrapper: MemoryRouter },
+    );
+
+    const sidebarItem = screen.getByRole(/group/i);
+
+    expect(sidebarItem).toHaveStyle({ pointerEvents: 'none', opacity: '0.6' });
```

Comment:
At the moment we are testing if the element gets the right style. Maybe a better solution is to check if it gets the disabled attribute? As I checked, toBeDisabled () doesn't work here, toHaveAttribute ('disabled') works instead, but you have to block this line for eslint. @KonradSzwarc which solution is better?